design/src/high-level/genre-analysis.md

```diff
@@ -13,67 +13,121 @@
 
 ## Defining characteristics
 
-- automated resource extraction
-- automation resource refinement
-- logistical challenges
-- micro-optimizations
+- automated resource extraction and refinement
+- logistical challenges driven by spatial constraints
+- optional optimization of existing systems (generally optimizing resource throughput)
 - gated technology progression
+- problems/puzzles/modules flow into each other (as opposed to puzzle games or zachlikes where they are explicitly independent)
+- partially self-directed goals
+- challenge can be overcome by spending time waiting
+- highly moddable
+- procedurally generatd world
+
+## Why is this genre fun?
+
+- puzzle like, but complete solutions are easy to achieve
+- optional optimizations on top of said solutions
+- satisfaction of watching your contraptions run
+- satisfaction of taming the wild
+- interesting, unexpected failures due to unexpected interactions or subtle bugs
```

Comment:
@TimJentzsch My guess is: *debugging* subtle failures in your design because of things you didn't understand at first is fun. Partially because it leads to a deeper understanding of how things actually fit together?

Reply:
Yep: bugs in the player's design, not in the game!

Reply:
```suggestion
- interesting, unexpected failures due to unexpected interactions or subtle bugs in the player's design
```